mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -219,33 +219,41 @@ class LinearRegression @Since("1.3.0") (@Since("1.3.0") override val uid: String
     }
 
     val yMean = ySummarizer.mean(0)
-    val yStd = math.sqrt(ySummarizer.variance(0))
+    var yStd = math.sqrt(ySummarizer.variance(0))
```

Comment:
Also, when `rawYStd == 0.0`, `standardization == true` and `regParam != 0.0`, the problem will be ill-defined. We may want to throw an exception.